docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3.6"
+services:
+  nginx:
+    image: nginx:1.17-alpine
+    volumes:
+      - ./../nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./../..:/var/www
+    ports:
+      - 5000:80
+    networks:
+      - internal
+    depends_on:
+      - php_fpm
+  php_fpm:
+    image: php_fpm
```

Comment:
This will result in an image name of `php_fpm` which could influence other projects, whats the reason for not using `engelsystem` or (better) `engelsystem-dev` or `engelsystem-dev-fpm`?

Reply:
I renamed the images